packages/lucide-preact/tests/createLucideIcon.spec.tsx

```diff
@@ -5,7 +5,7 @@ import { render } from '@testing-library/preact';
 
 describe('Using createLucideIcon', () => {
   it('should create a component from an iconNode', () => {
-    const AirVent = createLucideIcon('AirVent', airVent, 'air-vent');
+    const AirVent = createLucideIcon('air-vent', airVent);
```

Comment:
I think the test here should stay the same.
```suggestion
    const AirVent = createLucideIcon('AirVent', airVent);
```

You can add a second test for `air-vent` and `layout-2`.

Reply:
Agreed